components/datagrid/BodyRow.js

```diff
@@ -0,0 +1,40 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import Cell from './Cell';
+import Checkbox from '../checkbox';
+import Row from './Row';
+import cx from 'classnames';
+import theme from './theme.css';
+
+class BodyRow extends PureComponent {
+  static propTypes = {
+    className: PropTypes.string,
+    children: PropTypes.any,
+    onSelectionChange: PropTypes.func,
+    selectable: PropTypes.bool,
+    selected: PropTypes.bool,
+    sliceFrom: PropTypes.number,
+    sliceTo: PropTypes.number,
+  };
+
+  render() {
+    const { className, children, sliceFrom, sliceTo, onSelectionChange, selected, selectable, ...others } = this.props;
+
+    const childrenArray = Array.isArray(children) ? children : [children];
```

Comment:
This could be written like 
```js
const childrenArray = [...children];
```

And then maybe these 2 variables (`childrenArray` and `childrenSliced`) would not be necessary and can just write it inline?

Reply:
This doesn't work because `children` can be either an `object` or an `array`. The spread operator can not handle mixed types.

 :ok: by @driesd on 23/01/2018 15:35:10